cmd/describe.go

```diff
@@ -28,18 +27,26 @@ func init() {
 }
 
 var describeCmd = &cobra.Command{
-	Use:               "describe <name> [options]",
+	Use:               "describe [options]",
```

Comment:
Why shouldn't be I able to ask for description of function by name (which is not in cwd)?

Reply:
Basically for the same reasons I wrote in my [comment above](https://github.com/boson-project/faas/pull/125#discussion_r490348425) about `delete`. I think `describe` is another candidate for removal because how is `kn service describe <service-name>` any worse than `kn faas describe`? And again, if config moves upstream, then `kn service describe` === `kn faas describe`.

Reply:
>kn service describe === kn faas describe

Not exactly same, `faas` has slightly different formatting and also pulls info about subscriptions/triggers which `kn` doesn't IIRC.

Reply:
`kn` has separate sub-command for subscriptions/triggers.

Reply:
> Not exactly same, faas has slightly different formatting and it also pulls info about subscriptions/triggers which kn doesn't

Fair point. Let's discuss all of these commands in https://github.com/boson-project/faas/issues/126